### PR TITLE
Updated package.json

### PR DESCRIPTION
**Pull request by dangreenisrael**
Couldn’t get graphql up to 0.9.0, seems there was a breaking change

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,6 @@ coverage
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
+
+#IDE Stuff
+.idea
```
Comment (tothandras) [inline .gitignore]: this should be put in your global `gitignore`, mine looks like this:
```
$ cat ~/.gitignore_global
```
```
.sync-config.cson
*~
.DS_Store
.idea
.vscode
npm-debug.log
.DS_Store
.history
```

**Review by tothandras — CHANGES_REQUESTED** [review]

**Comment by dangreenisrael** [comment]
Removed.

**Comment by dangreenisrael** [comment]
Perhaps the breaking changes didn't kick in until the minor version update

**Comment by tothandras** [comment]
included by #206

**Pull request closed without merging**